src/interpreter.cpp

```diff
@@ -2868,7 +3135,7 @@ void nanny(DescriptorData *d, char *arg) {
         case YESNO_OTHER:
             string_to_output(d,
                              "\nPlease answer yes or no.\n"
-                             "&0&4&bDo you wish to keep this character (Y/n)?\n"
+                             "&0&4&bDo you wish to keep this character (y/n)?\n"
```

Comment:
Does this not default to YES anymore?

Reply:
It didn't default to yes before.  I just changed the capitalization on the "Y".